src/mobot_client/chat_strings.py

```diff
@@ -68,7 +69,7 @@ class ChatStrings:
 
     AIRDROP_RESPONSE = """We've sent you {amount} MOB (~{symbol}{value:.2f}). 
 
-Send us 0.01 MOB, and we'll send it back, plus more! 
+Send us 0.001 MOB, and we'll send it back, plus more! 
```

Comment:
Double checking on this - I think a few users reported sending the wrong amount today - is the amount checked for an exact match?

Reply:
This is completely arbitrary.